projects/packages/backup/src/js/components/Admin.js

```diff
@@ -317,13 +317,26 @@ const NoBackupCapabilities = () => {
 	const { tracks } = useAnalytics();
 	const [ priceAfter, setPriceAfter ] = useState( 0 );
 	const [ price, setPrice ] = useState( 0 );
+	const [ currencyCode, setCurrencyCode ] = useState( 'USD' );
+	const [ introOffer, setIntroOffer ] = useState( null );
 	const domain = useSelect( select => select( STORE_ID ).getCalypsoSlug(), [] );
 
 	useEffect( () => {
 		apiFetch( { path: '/jetpack/v4/backup-promoted-product-info' } ).then( res => {
+			setCurrencyCode( res.currency_code );
 			setPrice( res.cost / 12 );
+
 			if ( res.introductory_offer ) {
-				setPriceAfter( res.introductory_offer.cost_per_interval / 12 );
+				setIntroOffer( res.introductory_offer );
+				const {
+					cost_per_interval: costPerInterval,
+					interval_count: intervalCount,
+					interval_unit: intervalUnit,
+				} = res.introductory_offer;
+				const discountedPrice =
+					intervalCount === 1 && intervalUnit === 'month' ? costPerInterval : costPerInterval / 12;
```

Comment:
I wonder if we can go ahead and move towards a more flexible monthly price calculation function. 
Something like this maybe (pseudocode)

```js
// can pass intro offer or plan
function get_monthly_price( intro_offer ) {
    if ( intro_offer.intervalUnit === 'month' ){
        return intro_offer.costPerInterval / intro_offer.intervalCount
    } else if  ( intro_offer.intervalUnit === 'year' ) {
        return intro_offer.costPerInterval / ( intro_offer.intervalCount * 12 )
    }
}
```

That way we could handle things in a more robust way